handlers/handler-loadbalance/src/main/java/org/apache/servicecomb/loadbalance/LoadbalanceHandler.java

```diff
@@ -167,6 +176,23 @@ private void preCheck() {
 
   @Override
   public void handle(Invocation invocation, AsyncResponse asyncResp) throws Exception {
+    String endpointUri = invocation.getLocalContext(SERVICECOMB_SERVER_ENDPOINT);
+    if (endpointUri != null) {
+      String endpointRule = "[a-z]+://.+";
+      if (!endpointUri.matches(endpointRule)) {
+        throw new InvocationException(Status.BAD_REQUEST,
+            "the endpoint's format of the configuration is incorrect, e.g rest://127.0.0.1:8080");
+      }
+      URI formatUri = new URI(endpointUri);
+      Transport transport = CseContext.getInstance().getTransportManager().findTransport(formatUri.getScheme());
```

Comment:
do not use CseContext anymore, change to ScbEngine

Reply:
done